### PR TITLE
Style changes

### PR DESCRIPTION
**Pull request by paulkahle**
Typography - headers, statistic, quotation
Colors - New colors.  Left old colors for time being, but separated them and labeled them as deprecated in the docs.

### CODE REVIEW

File: src/stylesheets/vars/_colors.scss [hunk 1]
```diff
@@ -0,0 +1,198 @@
+/**
```
Comment (craigmichaelmartin) [inline src/stylesheets/vars/_colors.scss]: What is the difference between this file and `src/stylesheets/utils/_colors.scss`?

File: src/stylesheets/vars/_colors.scss [hunk 2]
```diff
@@ -0,0 +1,198 @@
...
+//
+$supre-colors: $colors;
+
+
+
+//
+// Create background colors to help with specificity issues
+// if supplied list is empty, it returns all background classes
+//
+@mixin appendBackgroundColorClasses($color-names...){
```
Comment (craigmichaelmartin) [inline src/stylesheets/vars/_colors.scss]: Are we using these mixins? It looks like these are generating "atomic" type classes.

File: src/stylesheets/vars/_colors.scss (hunk 1 above)
Comment (paulkahle) [reply src/stylesheets/vars/_colors.scss]: Looks like src/stylesheets/utils/_colors.scss was accidentally copied to this directory.  I'm removing it.

File: src/stylesheets/vars/_colors.scss (hunk 2 above)
Comment (paulkahle) [reply src/stylesheets/vars/_colors.scss]: These won't be used in the future, but will leave these until we have a chance to remove all the instances where there being used in Audri.

Comment (craigmichaelmartin) [reply src/stylesheets/vars/_colors.scss]: Sounds good! I thought they were new additions.

**Pull request merged**